<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="TYSS-Nandini" failures="1" tests="6" name="practice.SimActivation" time="0.017" errors="0" timestamp="12 Jul 2024 06:51:06 GMT" skipped="5">
  <testcase classname="practice.SimActivation" name="activatesim" time="0.001">
    <failure message="expected:&lt;[ ]&gt; but was:&lt;[Login]&gt;" type="junit.framework.ComparisonFailure">
      <![CDATA[junit.framework.ComparisonFailure: expected:<[ ]> but was:<[Login]>
at junit.framework.Assert.assertEquals(Assert.java:85)
at junit.framework.Assert.assertEquals(Assert.java:91)
at practice.SimActivation.activatesim(SimActivation.java:12)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.retryFailed(Invoker.java:839)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1010)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- activatesim -->
  <testcase classname="practice.SimActivation" name="activatesim" time="0.001">
    <skipped/>
  </testcase> <!-- activatesim -->
  <testcase classname="practice.SimActivation" name="activatesim" time="0.012">
    <skipped/>
  </testcase> <!-- activatesim -->
  <testcase classname="practice.SimActivation" name="activatesim" time="0.002">
    <skipped/>
  </testcase> <!-- activatesim -->
  <testcase classname="practice.SimActivation" name="activatesim" time="0.000">
    <skipped/>
  </testcase> <!-- activatesim -->
  <testcase classname="practice.SimActivation" name="activatesim" time="0.001">
    <skipped/>
  </testcase> <!-- activatesim -->
</testsuite> <!-- practice.SimActivation -->
